docs/classic/go.md

```diff
@@ -15,21 +15,21 @@ agent-labels: "agents= ['black_0', 'white_0']"
 
 
 
-Go is a board game with 2 players, black and white. The black player starts by placing a black stone at an empty board intersection. The white player follows with the purpuse of surrounding more territory than the opponent or capturing the opponent's stones. The game ends if both players sequentially decide to pass. Refer to [Wikipedia](https://en.wikipedia.org/wiki/Go_(game)) for additional details about the game.
+Go is a board game with 2 players, black and white. The black player starts by placing a black stone at an empty board intersection. The white player follows by placing a stone of their own, aiming to either surround more territory than their opponent or capture the opponent's stones. The game ends if both players sequentially decide to pass. Refer to [Wikipedia](https://en.wikipedia.org/wiki/Go_(game)) for additional details about the game.
```

Comment:
"refer to" sounds off to me

Reply:
I think its correct

Reply:
I don't mean it's grammatically rude, I mean it feels mildly disparaging to the reader

Reply:
Oh, I could put the "for additional details about the game." at the start of the sentence? sounds a little more firendly

Reply:
Honestly pull the wikipedia link all together (do a ctrl f and pull others if they're still in)

Reply:
Ok, I don't think we link to Wikipedia much so might as well remove them